conference/cart.py

```diff
@@ -256,11 +259,16 @@ def send_order_confirmation_email(order: Order, current_site) -> None:
 
 
 class TicketType:
+    """
+    NOTE(artcz): This should be in sync with assopy/models.py::ORDER_TYPE
+    """
     personal = "personal"
     company = "company"
     student = "student"
 
-    ALL = [personal, company, student]
+    CONFERENCE_OR_TRAINING = [personal, company, student]
+    OTHER = "other"
```

Comment:
This feels inconsistent. Why is OTHER caps, alternatively, why is OTHER not a list?

Reply:
Good catch - I'll lowercase `other` to keep things consistent.